Tests/CucumberSwiftTests/Reporter/Schema.json

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
Is this a JSONSchema representation of the proper Cucumber JSON report? Love this, thank you!